### ISSUE TITLE
Add plotly support

### ISSUE THREAD

**Issue by ChRen95** [pyroll-project/pyroll-report#17]
We should add the support for plotly in the report. Also we should reconsider what plots should be drawn by default using the plotly library

**Comment by axtimhaus** [comment]
Was not completed anywhere. Why did you close it?

**Comment by ChRen95** [comment]
Is this finished @axtimhaus ?

**Comment by axtimhaus** [comment]
No, there is no progress on. The report currently plots all thing s with matplotlib
